apis/core/lib/domain/dimension-metadata/DimensionMetadata.ts

```diff
@@ -0,0 +1,46 @@
+import { Constructor } from '@tpluscode/rdfine'
```

Comment:
Uh, oh., I created the same model in `apis/core/lib/domain/dimension` directory. Sorry 😊 

Please merge `dimension` and `dimension-metadata`. Flip a coin to decide on the name :)